build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
 	implementation 'org.springframework.boot:spring-boot-starter-data-rest'
 	
+	compile ('com.h2database:h2')
```

Comment:
this should be testCompile

https://www.tutorialspoint.com/gradle/gradle_dependency_management.htm#:~:text=Test%20Compile%20%E2%88%92%20The%20dependencies%20required,runtime%20and%20test%20compile%20dependencies.

Reply:
Implemented.